src/Microsoft.Sbom.Api/Executors/Spdx2SerializationStrategy.cs

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Sbom.Api.Entities;
+using Microsoft.Sbom.Api.Providers;
+using Microsoft.Sbom.Extensions;
+
+namespace Microsoft.Sbom.Api.Workflows.Helpers;
+
+/// <summary>
+/// Serialization methods for SPDX 2.2.
+/// </summary>
+public class Spdx2SerializationStrategy : IJsonSerializationStrategy
```

Comment:
Let's make `JsonSerializationStrategyFactory`, `Spdx3SerializationStrategy`, `Spdx2SerializationStrategy`, and `IJsonSerializationStrategy` all `internal` unless we need them for unit tests. They shouldn't be part of the public API.
```suggestion
internal class Spdx2SerializationStrategy : IJsonSerializationStrategy
```